tools/python/brownie/great_ape_safe/ape_api/across.py

```diff
@@ -0,0 +1,29 @@
+import requests
+from helpers.addresses import registry, r
+
+
+
+class Across:
+    def __init__(self, safe):
+        self.safe = safe
+
+        # See https://docs.across.to/v/developer-docs/developers/across-api#calculating-suggested-fees
+        # Any chain's token address can be used
```

Comment:
I'd recommend to always specify the originChainId in the request, so there's no ambiguity about where you're bridging from. We have seen issues with DAI on Arbitrum and Optimism, because they share the same token address on those chains.